### PR TITLE
Seeding scripts

### LINKED ISSUE #2239: Create script to apply only new data from fixtures
When deploying it's not really possible to use `loaddata` and only add new entries.

Create a script that will add new entries from selected fixtures; it should accept a table column by which existence of an entry should be determined.

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
This fixes #2239 by providing an alternative to fixtures from Django.

Seeding scripts give more control to the developer. This way it was possible to implement a check before given entity gets created.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203911669015361

### CODE REVIEW

File: amy/workshops/utils/seeding.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    def _info(msg: str) -> None:
+        if logger:
+            logger.info(msg)
+
+    class_name = model_class.__name__
+
+    _info(f"Start of {class_name} deprecation.")
+
+    for i, model_id in enumerate(model_id_list):
+        if not model_class._default_manager.filter({lookup_field: model_id}).exists():
```
Comment (elichad) [inline amy/workshops/utils/seeding.py:52]:
```suggestion
        if not model_class._default_manager.filter(**{lookup_field: model_id}).exists():
```

File: amy/workshops/utils/seeding.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            _info(f"{i} {class_name} <{model_id}> doesn't exist, skipping.")
+            continue
+
+        _info(f"{i} {class_name} <{model_id}> exists, removing.")
+        model_class._default_manager.filter({lookup_field: model_id}).delete()
```
Comment (elichad) [inline amy/workshops/utils/seeding.py:57]:
```suggestion
        model_class._default_manager.filter(**{lookup_field: model_id}).delete()
```

**Review by elichad — CHANGES_REQUESTED** [review]
The `deprecate_models` method doesn't work for me if I add names to `DEPRECATED_X` variables - suggestions should fix it. Looks good otherwise.

Does the seeding get run automatically during the deployment process?

File: amy/workshops/utils/seeding.py (hunk 1 above)
Comment (pbanaszkiewicz) [reply amy/workshops/utils/seeding.py:52]: Good catch! Thanks, fixing now.

**Review by elichad — APPROVED** [review]

**Pull request merged**